legal-api/src/legal_api/services/authz.py

```diff
@@ -95,96 +107,441 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
+                }
+            },
+            'changeOfAddress': {
+                'legalTypes': ['CP', 'BEN', 'BC', 'ULC', 'CC'],
+                'blockerChecks': {
+                    'business': [BusinessBlocker.DEFAULT]
+                }
+            },
+            'changeOfDirectors': {
+                'legalTypes': ['CP', 'BEN', 'BC', 'ULC', 'CC'],
+                'blockerChecks': {
+                    'business': [BusinessBlocker.DEFAULT]
+                }
+            },
+            'changeOfRegistration': {
+                'legalTypes': ['SP', 'GP'],
+                'blockerChecks': {
+                    'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],
+                    'business': [BusinessBlocker.DEFAULT]
+                }
+            },
+            'consentContinuationOut': {
+                'legalTypes': ['BC', 'BEN', 'CC', 'ULC'],
+                'blockerChecks': {
+                    'business': [BusinessBlocker.DEFAULT]
+                }
+            },
+            'conversion': {
+                'legalTypes': ['SP', 'GP']
+            },
+            'correction': {
+                'legalTypes': ['CP', 'BEN', 'SP', 'GP', 'BC', 'ULC', 'CC'],
+                'blockerChecks': {
+                    'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],
+                    'business': [BusinessBlocker.DEFAULT]
+                }
+            },
+            'courtOrder': {
+                'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+            },
+            'dissolution': {
+                'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC', 'SP', 'GP'],
+                'blockerChecks': {
+                    'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],
+                    'business': [BusinessBlocker.DEFAULT]
+                }
+            },
+            'incorporationApplication': {
+                'legalTypes': ['CP', 'BC', 'BEN', 'ULC', 'CC'],
+                'businessExists': False  # only show filing when providing allowable filings not specific to a business
+            },
+            'registrarsNotation': {
+                'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+                'blockerChecks': {
+                    'business': [BusinessBlocker.DEFAULT]
+                }
+            },
+            'registrarsOrder': {
+                'legalTypes':  ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC']
+            },
+            'registration': {
+                'legalTypes': ['SP', 'GP'],
+                'businessExists': False  # only show filing when providing allowable filings not specific to a business
+            },
+            'specialResolution': {
+                'legalTypes': ['CP']
+            },
+            'transition': {
+                'legalTypes': ['BC', 'BEN', 'CC', 'ULC']
+            },
             'restoration': {
```

Comment:
How about keeping this list sorted alphabetically (to make is easier to find a particular filing)?

Reply:
yes, good call